workbench-core/base/src/utilities/validatorHelper.ts

```diff
@@ -27,6 +37,22 @@ z.ZodString.prototype.swbId = function (prefix: string): ZodString {
   return this.regex(uuidWithLowercasePrefixRegExp(prefix), { message: 'Invalid ID' });
 };
 
+z.ZodString.prototype.nonHTML = function (): ZodString {
+  return this.regex(nonHtmlRegExp(), { message: 'Input contains HTML' });
+};
+
+z.ZodString.prototype.swbName = function (): ZodString {
+  return this.max(swbNameMaxLength, {
+    message: `Input must be less than ${swbNameMaxLength} characters`
+  }).regex(swbNameRegExp(), { message: 'invalid input characters' });
```

Comment:
Can we specify the valid characters
e.g. `Must contain only letters, numbers, hyphens, underscores, and periods`